clean_project/test_data/scriptdb.sql

```diff
@@ -0,0 +1,13 @@
+-- Create a database with a specific owner
+CREATE DATABASE index_db OWNER postgres;
+-- Create user with a specific option
+CREATE USER gianluca [ [ WITH ] option [ ... ] ]
```

Comment:
what is this? this is not a command you should use, this is the format of the command...